src/index.js

```diff
@@ -17,6 +17,19 @@ const loadDeps = () => {
 };
 
 module.exports = {
+  config: {
+    defaultConfig: {
+      title: 'Default Config Path',
+      type: 'string',
+      default: atom.getConfigDirPath(),
```

Comment:
I would just leave this blank, if a user wants to fill this in they can enter the path to wherever it should be themselves. The default value as is is wrong and just encourages people to put non-Atom configuration files in there.